### ISSUE TITLE
WHen the initial database unlock page is shrunken to its minimum height the key file feild is shrunk to unusable. 

### ISSUE THREAD

**Issue by dpbaker57** [keepassxreboot/keepassxc#10881]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. go to database unlock page for a database with a key file
2. shrink the window to the vertical minimum
3. Try to do anything in the key file name

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
THe name should be editable

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
Too thin to do anything

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.7.8
Revision: f6757d3

Qt 5.15.11
Debugging mode is disabled.

Operating system: Windows 10 Version 2009
CPU architecture: x86_64
Kernel: winnt 10.0.19045

Enabled extensions:
- Auto-Type
- Browser Integration
- Passkeys
- SSH Agent
- KeeShare
- YubiKey
- Quick Unlock

Cryptographic libraries:
- Botan 3.1.1

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows
Desktop Env: Gnome/KDE/XFCE/Mate/Cinnamon
Windowing System: X11/Wayland

**Comment by droidmonkey** [comment]
Duplicate report

**Comment by dpbaker57** [comment]
I guess I don't understand what "not planned" means? You won't fix it or it
is just not on the current schedule or we will accept a fix if someone
codes it but not until then.
*Darryl Baker*
